Source/Details/ASDataController.h

```diff
@@ -65,6 +65,11 @@ extern NSString * const ASCollectionInvalidUpdateException;
  */
 - (NSUInteger)numberOfSectionsInDataController:(ASDataController *)dataController;
 
+/**
+ Fetch the rect of a cell for at a given index path
+ */
+- (CGRect)dataController:(ASDataController *)dataController rectForRowAtIndexPath:(NSIndexPath *)indexPath;
```

Comment:
To avoid confusion, we should make this method more clearly related to "what is visible" and we should support supplementary elements too. Maybe:

`- (CGSize)dataController:(ASDataController *)dataController committedSizeForElement:(ASCollectionElement *)element`